### PR TITLE
Mre doc edits

### PR DESCRIPTION
**Pull request by MaureenElsberry**
Small edits for the Arrow Fx, async, and polymorphism readmes

### CODE REVIEW

File: modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md [hunk 1]
```diff
@@ -158,10 +157,10 @@ The following combinators illustrate how the Functor hierarchy functions are poi
 | MonadDefer.delay    | `IO.delay { 1 }` | `effect { 1 }` |
 | MonadDefer.defer    | `IO.defer { IO { 1 } }` | `effect { 1 }` |
 
-This is in general true for effectful data types that are non-commutative. 
-Note that implicit CPS style with non-blocking direct binding has the disadvantage that for non-commutative monads where the order of effects matter you can't apply substitution based on referential transparency.
+This is, in general, true for effectful data types that are non-commutative.
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md]: I would add a new header here that reads:

`Effects with non-commutative monads`

Comment (pakoito) [reply modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md]: Also, the line should read `are commutative.`

Comment (raulraja) [reply modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md]: Addressed as part of the last master merge

**Comment by pakoito** [comment]
I had fixed a bunch of links already on master, this diff has smashed over them 🙈

For example,

```
[`parMapN`](#-parmapn-)
```

is a regression.

**Comment by raulraja** [comment]
Let's get rid of those TOC at the top. They are a burden to maintain unless we automate it and then they would not be there in the first place.

**Pull request merged**